sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.12.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.12.0 (2024-05-06)
```

Comment:
Holding off on a Core GA until we remove the unnecessary public methods on `RawResponse`:
https://github.com/Azure/azure-sdk-for-cpp/pull/5600